controllers/webhooks/networking/cfroute_validator.go

```diff
@@ -101,14 +101,31 @@ func (v *CFRouteValidator) ValidateUpdate(ctx context.Context, oldObj, obj runti
 		return apierrors.NewBadRequest(fmt.Sprintf("expected a CFRoute but got a %T", obj))
 	}
 
-	if route.Spec.Host == oldRoute.Spec.Host &&
-		route.Spec.DomainRef.String() == oldRoute.Spec.DomainRef.String() &&
-		route.Spec.Path == oldRoute.Spec.Path &&
-		len(route.Finalizers) < len(oldRoute.Finalizers) {
-		return nil // we're removing a finalizer to delete this route, so we don't want to block deletion
+	immutableError := webhooks.ValidationError{
+		Type: webhooks.ImmutableFieldErrorType,
 	}
 
-	domain, err := v.validateRoute(ctx, route)
+	if route.Spec.Host != oldRoute.Spec.Host {
+		immutableError.Message = fmt.Sprintf(webhooks.ImmutableFieldErrorMessageTemplate, "Host")
```

Comment:
Should these errors be the full path to the field like the domain ones? ie) CFRoute.Spec.Host